tests/test_other.py

```diff
@@ -10233,6 +10233,63 @@ def test_split_main_module(self):
     self.assertIn('Hello from main!', result)
     self.assertIn('Hello from lib!', result)
 
+  def test_split_module_ex(self):
+    initialTableSize = 5632
+    self.set_setting('SPLIT_MODULE')
+    self.set_setting('WASM', 1)
+    self.set_setting('WASM_BIGINT')
+    self.set_setting('EVAL_CTORS', 0)
+    self.set_setting('INLINING_LIMIT', 1)
+    self.set_setting('FORCE_FILESYSTEM', 1)
+    self.set_setting('LEGACY_GL_EMULATION', 0)
+    self.set_setting('DISABLE_EXCEPTION_CATCHING', 0)
+    self.set_setting('ALLOW_MEMORY_GROWTH', 1)
+    self.set_setting('NO_EXIT_RUNTIME', 0)
+    self.set_setting('GL_TRACK_ERRORS', 0)
+    self.set_setting('WASM_OBJECT_FILES', 0)
+    self.set_setting('INCLUDE_FULL_LIBRARY', 1) # https://github.com/emscripten-core/emscripten/issues/13794
```

Comment:
Remove this line, we have tests for this elsewhere now I believe

Reply:
My understanding was that the tests included in https://github.com/emscripten-core/emscripten/pull/13863 should cover the #13794 issue.   If I'm wrong then maybe I'm not fully understanding the #13794 issue or what fixed it.   But testing of INCLUDE_FULL_LIBRARY can be done in isolation I believe.  I don't think #13794 relates to SPLIT_MODULE.

Looking back at #13863, I do see your comment about how it didn't immediately fix your issue, so I will go back and try to find out exactly what change did fix the issue, and come up with a precise test for that. I